doc/schema.md

```diff
@@ -1,6 +1,7 @@
 # Index Schema
 
-The index is stored at a single RocksDB database using the following column families:
+The index is stored at a single RocksDB database using the following column families.
+Most of the data is stored in key-only DB rows (i.e. having empty values).
```

Comment:
I probably didn't used the right terminology... will try to explain what I've meant:

Since our indices may map a single hash prefix to multiple block heights (due to an address being funded in multiple blocks or just a hash prefix collision), we need to store both the hash prefix and the block height together (as a single RocksDB key).
This trick allows us to use prefix scan to get all block heights that correspond to a specific scripthash/txid prefix.

In addition, those rows are immutable - i.e. after they are written there is no need to update them.

Therefore, there is no need for those index rows to use RocksDB values:
https://github.com/romanz/electrs/blob/fe1bfe3bc1d67a08feab56c242ce320804ed62de/src/db.rs#L169-L182

Reply:
I assume RocksDb has some optimization for this case, right?

Reply:
IIUC, the overhead is a single byte for encoding the value's zero length.
https://github.com/google/leveldb/blob/master/doc/table_format.md
https://github.com/facebook/rocksdb/blob/beed86473a078f58e6a8f214068a142bfc53d6aa/table/block_based/block_builder.cc#L24
https://github.com/facebook/rocksdb/blob/beed86473a078f58e6a8f214068a142bfc53d6aa/util/coding.cc#L28